assets/styles/settings/_config.scss

```diff
@@ -43,13 +43,18 @@ $easing: $Power2EaseOut;
 // Spacing Units
 // =============================================================================
 $unit:       60px;
-$unit-small: 30px;
+$unit-small: 20px;
 
 // Container
 // ==========================================================================
 $container-width: 2000px;
```

Comment:
On aurait pu besoin de `$container-width` je pense? Je le ditch toujours de mon `o-container`

Reply:
@devenini Was this question answered?

Reply:
Yes! `$container-width` was removed.